Classes/Emogrifier.php

```diff
@@ -418,6 +421,37 @@ public function removeUnprocessableHtmlTag($tagName)
         }
     }
 
+     /**
+      * This removes styles from your email that contain display:none.
+      * We need to look for display:none, but we need to do a case-insensitive search. Since DOMDocument only
+      * supports XPath 1.0, lower-case() isn't available to us. We've thus far only set attributes to lowercase,
+      * not attribute values. Consequently, we need to translate() the letters that would be in 'NONE' ("NOE")
+      * to lowercase.
+      *
+      * @param \DOMXPath $xpath
+      *
+      * @return void
+      */
+    private function removeInvisibleNodes(\DOMXPath $xpath)
+    {
+        $nodesWithStyleDisplayNone = $xpath->query(
+            '//*[contains(translate(translate(@style," ",""),"NOE","noe"),"display:none")]'
+        );
+
```

Comment:
Please remove this blank line so that the guard clause is a single visible block.